dashboard/dashboard/templates/registration/change_password.html

```diff
@@ -0,0 +1,37 @@
+{% extends "racetrack/base.html" %}
+
+{% block content %}
+
+<div class="container">
+<div class="card">
+<div class="card-body">
+
+    <h5 class="card-title">Change Password</h5>
+
+    <form method="post">
+        {% csrf_token %}
+
+        {% for field in form %}
+            <div class="mb-3">
+                <label class="form-label">{{ field.label_tag }}</label>
+                {{ field }}
+                {% if field.help_text %}
+                <div class="form-text">{{ field.help_text|safe }}</div>
+                {% endif %}
+
+                {% for error in field.errors %}
+                    <div class="invalid-feedback d-block">
+                    {{ error|escape }}
+                    </div>
+                {% endfor %}
+            </div>
+        {% endfor %}
+
+        <button type="submit" class="btn btn-primary">Change Password</button>
```

Comment:
Recently at workshop I learned about zxcvbn - it's a password strength validation tool, to encourage users to use more secure passwords. Because with simple rules like use big letter, use number, use special char, we end up with passwords like "Password1!". Hackers know this, and setup their brute force tools accordingly, and passwords can be cracked in a matter of minutes or hours (instead of days or months, like it should be). So https://github.com/dropbox/zxcvbn was invented to improve on that, and https://github.com/Pawamoy/django-zxcvbn-password is django plugin (looking reasonably supported) with nice bar showing how long it will take to crack used password. So it's a UI eye-candy, that encourages user to choose more secure password. If you have the time, might be worth installing it (perhaps separate PR).

![Django zxcvbn](https://cloud.githubusercontent.com/assets/3999221/23079032/5ae1513a-f54b-11e6-9d66-90660ad5fb2d.png)

Reply:
Cool, it might be noteworthy. Though in this case, having so many restrictive (annoying :smile: ) rules provided by Django: "password must contain at least 8 characters, can't be a commonly used password, can't be entirely numeric" there's a higher chance of ending up with the strong password either way.